plugins/modules/files/json_file.py

```diff
@@ -0,0 +1,540 @@
+        those provided in arguments.
+    type: str
+    choices: [ present, absent, identic ]
+    default: present
+  list_diff_type:
+    description:
+      - This describes how the list are compared.
+      - If set to C(value), it checks if a value is present in the list
+        elements.  With this, you can't check a list element in another list
+        element.  Only the first level of a list can be checked.
+      - If set to C(index), the value in a list are compared by their index.
+        With this, you can check an element in another list, but you make sure
+        that the expected value is at this position in the list.
+    type: str
+    choices: [ value, index ]
+    default: value
+  value:
+    description:
+      - The values that you want to compare with the one present in the json
+        file.
+    type: dict
+    required: true
+  create:
+    description:
+      - If set to C(no), the module will fail if the json file not exists.
+        Else, the json file will be created.
+    type: bool
+    default: false
+  diff_on_value:
+    description:
+      - This describes how check the if it has a difference between the datas
+        present in the json file and the expected values. This will also
+        impact the output in diff mode (in a case you get indents or lines
+        breaks in the diff will in the other you only gets values).
+      - If set to C(true), it checks the difference by only comparing the
+        values of elements in the json file and in the expected datas. In this
+        case, change indentation or sort keys will not be considered as a change
+        on the file will the values are the sames (just like any other
+        difference in json file unrelated to values of elements like line breaks
+        or comments).
+      - If set to C(false), it checks the difference by comparing the lines
+        output with the actual lines in the json file. In this case, all the
+        change on the indentation or on the sort keys will be considered as a
+        change on the file even if it has no difference on the values (if a
+        comment, or an empty line break is present in the file, a change will
+        be detected).
+    type: bool
+    default: true
+  backup:
+    description:
+      - Create a backup file including the timestamp information so you can get
+        the original file back if you somehow clobbered it incorrectly.
+    type: bool
+    default: no
+  indent:
+    description:
+      - Set the indentation for elements in the json file.
+        See U(https://docs.python.org/3/library/json.html#basic-usage).
+    type: int
+    default: 4
+  sort_keys:
+    description:
+      - If set to C(true), the values in the will sorted by their keys.
+        See U(https://docs.python.org/3/library/json.html#basic-usage).
+    type: bool
+    default: false
+author:
+    - DEMAREST Maxime (Indelog) <maxime@indelog.fr>
```

Comment:
```suggestion
    - DEMAREST Maxime (@indelog) <maxime@indelog.fr>
```
Your GitHub username is `indelog`, not `Indelog`.